### PR TITLE
[TAS-1573] 💄 Add sorting and filtering UI in /listing

### PR DESCRIPTION
**Pull request by AuroraHuang22**
- [x] filter section
- [ ] queries
- [x] header
- [x] scroll to top 
- [x] QA section

截圖 2024-05-13 上午10 50 25
截圖 2024-05-13 上午10 50 34
截圖 2024-05-13 上午10 50 45
截圖 2024-05-13 上午10 50 50
截圖 2024-05-13 上午10 53 10

### CODE REVIEW

File: src/components/ListingPage/MobileFilterSection.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="flex flex-col">
+    <Label
+      :text="$t('listing_page_filter')"
+      align="center"
+      class="mb-[12px] text-[18px] font-600"
+    />
+    <div class="w-full border-b-[1px] border-shade-gray my-[8px]" />
+    <ListingPageFilterSection
```
Comment (nwingt) [inline src/components/ListingPage/MobileFilterSection.vue:9]: Add a prop for mobile and dont use CSS breakpoint
```suggestion
    <ListingPageFilterSection
      :is-mobile="true"
```

File: src/components/ListingPage/MobileFilterSection.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      :selected-price="selectedPrice"
+      :selected-language="selectedLanguage"
+      @handle-click-confirm="handleConfirm"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'MobileSortingSection',
```
Comment (nwingt) [inline src/components/ListingPage/MobileFilterSection.vue:20]:
```suggestion
  name: 'MobileFilterSection',
```

File: src/components/ListingPage/QASection.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    v-bind="$attrs"
+    class="px-[20px] py-[24px] rounded-[12px] bg-white shadow-md"
+  >
+    <Label preset="h3" class="text-like-green mb-[16px]">常見問題</Label>
```
Comment (nwingt) [inline src/components/ListingPage/QASection.vue:6]: Translate

File: src/pages/listing/index.vue [hunk 4]
```diff
@@ -0,0 +1,296 @@
+<template>
+  <Page
+    :class="[
+      'relative',
+      'overflow-x-hidden',
+      'max-w-[1924px]',
```
[the remark this answers is not in the document]
Comment (AuroraHuang22) [reply src/pages/listing/index.vue:6]: [💄 Add margin](https://github.com/likecoin/liker-land/pull/1612/commits/52a0f40d1a4af98801d84fa5677756e9737b375c)

File: src/pages/listing/index.vue [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+      :scrollable-wrapper-classes="[
+        '!items-end',
+        'phone:pb-[12px]',
+        'phone:px-[12px]',
+        'phoneLg:pb-[0px]',
+        'phoneLg:px-[0px]',
+        'py-[80px]',
+      ]"
+      @close="handleCloseDialog"
+    >
```
[the remark this answers is not in the document]
Comment (AuroraHuang22) [reply src/pages/listing/index.vue]: [💄 Add <ListingPageDialog/>](https://github.com/likecoin/liker-land/pull/1612/commits/389a25a0a96802ab4170e10d5af15e91e5e1c63c)

File: src/components/ListingPage/MobileFilterSection.vue (hunk 1 above)
Comment (AuroraHuang22) [reply src/components/ListingPage/MobileFilterSection.vue:9]: [💄 Add ListingPageDialog](https://github.com/likecoin/liker-land/pull/1612/commits/389a25a0a96802ab4170e10d5af15e91e5e1c63c)

File: src/components/ListingPage/MobileFilterSection.vue (hunk 2 above)
Comment (AuroraHuang22) [reply src/components/ListingPage/MobileFilterSection.vue]: [🎨 Correct component name](https://github.com/likecoin/liker-land/pull/1612/commits/8824ec607392dc21afafea9c01e4e478d105cb47)

File: src/components/ListingPage/QASection.vue (hunk 3 above)
Comment (AuroraHuang22) [reply src/components/ListingPage/QASection.vue]: [🌐 Add i18n](https://github.com/likecoin/liker-land/pull/1612/commits/cf6e188791c8f57ad41cdaf6cc5727066efdbf20)

**Pull request merged**